operator/main.go

```diff
@@ -254,7 +254,10 @@ func runOperator() {
 				if identity == operatorID {
 					log.Info("Leading the operator HA deployment")
 				} else {
-					log.WithField("operator-id", operatorID).Infof("Operator with ID %q elected as new leader", identity)
+					log.WithFields(logrus.Fields{
+						"newLeader": operatorID,
+						"identity":  identity,
```

Comment:
This should be the ID of the operator in context, so `"operatorID": opeatorID`. The term identity is not very clear we can have `operatorID` instead.

Reply:
@fristonio can you check now, made the change